src/components/Card.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+const cardStyle = makeStyles(theme => ({
+  root: {
+    padding: theme.spacing(3, 2)
+  }
+}));
+
+const card = props => {
+  const classes = cardStyle();
+  const { title, body, type } = props.content;
```

Comment:
This can be restructured on line 14 at the function declaration:
```js
  const Card = ({ content: { title, body, type } } = { }) => { 
  ...
```